### DISCUSSION TITLE
Details not shown when not in place

### DISCUSSION THREAD

**Discussion by Antonio-52** [streetcomplete/StreetComplete#5321] [Q&A] [answered]
In StreetComplete I cannot see details of a place far from where I am.
Why? 
Is it to avoid compiling errors when not in place?
Is it possible to add a setting unable/disable to allow editing of far but well known places?
Thank you

**Comment by HolgerJeromin** [comment, marked as answer]
You can scroll to any place and use "download here" from the menu.
